afs/afs-core/src/test/java/com/powsybl/afs/AfsBaseTest.java

```diff
@@ -33,12 +33,13 @@
 
     @Before
     public void setup() {
-        storage = new DefaultListenableAppStorage(MapDbAppStorage.createMem("mem"));
+        storage = MapDbAppStorage.createMem("mem");
 
         ComputationManager computationManager = Mockito.mock(ComputationManager.class);
         afs = new AppFileSystem("mem", true, storage);
         ad = new AppData(computationManager, computationManager, Collections.singletonList(computationManager1 -> Collections.singletonList(afs)),
                 Collections.emptyList(), Collections.singletonList(new FooFileExtension()), Collections.emptyList());
+        storage.setEventStore(ad.getEventsStore());
```

Comment:
I think it would be cleaner to pass event store instance through AppFileSystemProviderContext in the AppData class and setting it directly in MapDbStorage constructor instead of using AppStorage.setEventStore.